Houdini/src/Controller/HoudiniController.php

```diff
@@ -173,4 +182,29 @@ public function identify(Request $request)
             return new Response($e->getMessage(), 500);
         }
     }
+
+    /**
+     * imagemagick will sometimes leave tmp files behind, this will clean them up.
+     * Be careful about when you call this, else you could delete work that's in
+     * progress.
+     */
+     protected function cleanupTmpFiles() {
+
+        // If there's are magick-* or php* files in /tmp, clean them up.
+        // We could just do this, but for now let's add some logging.
+        //   array_map('unlink', glob('/tmp/magick-*'));
+        //   array_map('unlink', glob('/tmp/php*'));
+        try {
+            foreach (glob('/tmp/magick-*') as $filename) {
+                $this->log->info("removing file $filename");
+                unlink($filename);
+            }
+            foreach (glob('/tmp/php*') as $filename) {
```

Comment:
This might be too much or unnecessary.  Not sure if the php tmp files get left behind.

Reply:
A couple of question about the `php*` files:
* do we know how large they are?  that is, what is the consequence of leaving them alone?  if they are small, they just contribute to the overall clutter of `/tmp` but are otherwise harmless.
* do we know if the `php*` files are a byproduct of imagemagick itself, or are they a byproduct of the PHP engine?  If PHP creates these files, how do we know that they are a direct consequence to imagemagick operation, and not critical to the PHP engine operation?

I'm thinking if the `php*` files are small, we just get rid of the `magick-*` files and leave the `php*` be.  Thoughts?

Reply:
Yeah, that's what I'm thinking as well after reading about php and tmp files.  I know the algorithm is creating a tmp php file in the code that executes the commands [here](https://github.com/Islandora/Crayfish-Commons/blob/2.x/src/CmdExecuteService.php#L55).  That being said, I'm not sure if other parts of the PHP engine are doing that too, and I can see how it could be dangerous to clean them up w/o knowing.

From what I've read, tmp php files created this way get cleaned up when the script stops running, so they are probably not an issue for us, overall. I can take those lines out that cleanup `php*` files.